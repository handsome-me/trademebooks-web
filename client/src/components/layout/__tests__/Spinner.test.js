import React from 'react'
import renderer from 'react-test-renderer'

import Spinner from '../Spinner'

describe('Component: Spinner', () => {
  it('should match the snapshot', () => {
    const snapshot = renderer.create(<Spinner />).toJSON()

    expect(snapshot).toMatchSnapshot()
  })
})
