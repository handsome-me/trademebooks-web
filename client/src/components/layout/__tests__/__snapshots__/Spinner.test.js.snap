// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Spinner should match the snapshot 1`] = `
<img
  alt="Loading..."
  src="spinner.gif"
  style={
    Object {
      "display": "block",
      "margin": "auto",
      "width": "200px",
    }
  }
/>
`;
